src/godot_fmod.h

```diff
@@ -44,13 +44,13 @@ namespace godot {
 #define GODOT_LOG(level, message)\
     switch (level) {\
         case 0:\
-            Godot::print(message);\
+            UtilityFunctions::print(message);\
             break;\
         case 1:\
-            Godot::print_warning(message, BOOST_CURRENT_FUNCTION, __FILE__, __LINE__);\
+            UtilityFunctions::print(message, BOOST_CURRENT_FUNCTION, __FILE__, __LINE__);\
```

Comment:
⚠ CAREFUL ⚠

It seems there is no warn print any longer? Maybe instead, we need to use the "push warn" method?

Reply:
Used push for now. Let's keep an eye on it.